.eslintrc.json

```diff
@@ -56,6 +52,7 @@
       ],
       "plugins": ["@typescript-eslint"],
       "rules": {
+        "import/named": 0,
```

Comment:
Why is this not a 2 like the others?

Reply:
Good question: because TypeScript provides this check automatically.  `plugin:import/typescript` disables it (i.e uses 0), but I had to re-enable it for the 'default' linting because it runs on .js

(yeah, it's weird that `plugin:import/typescript` is applied to .js files, but it seems to Just Work).